app/profile/page.tsx

```diff
@@ -0,0 +1,303 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+import { Suspense, useMemo } from 'react';
+import { formatEther, isAddress } from 'viem';
+import {
+  calcSellAmountInCollateral,
+  formatValueWithFixedDecimals,
+  getGnosisAddressExplorerLink,
+  shortenAddress,
+} from '@/utils';
+import { TabBody, TabGroup, TabHeader, Tag } from '@swapr/ui';
+import {
+  Market,
+  Position,
+  tradesCollateralAmountUSDSpent,
+  tradesOutcomeBalance,
+  tradesVolume,
+} from '@/entities';
+import { getUser } from '@/queries/conditional-tokens';
+import { UserPositionComplete } from '../my-bets/page';
+import { getUserPositionsComplete } from '@/queries/omen';
+import { useQuery } from '@tanstack/react-query';
+import { fromUnixTime, format } from 'date-fns';
+import {
+  BetsListPanel,
+  BetsListPanelProps,
+  BetsListTab,
+  ProfileCardBet,
+} from '@/app/components';
+import { cx } from 'class-variance-authority';
+import Image from 'next/image';
+import { getAIAgents } from '@/queries/dune';
+
+export default function ProfilePage() {
+  return (
+    <Suspense>
+      <Profile />
+    </Suspense>
+  );
+}
+
+const Profile = () => {
+  const searchParams = useSearchParams();
+
+  const address = searchParams.get('address')?.toLocaleLowerCase();
+
+  const { data: userInfo } = useQuery({
+    queryKey: ['getUser', address],
+    queryFn: async () => {
+      if (!address) return;
+      return getUser({ id: address });
+    },
+    enabled: !!address,
+  });
+
+  const { data: userPositions, isLoading } = useQuery<UserPositionComplete[]>({
+    queryKey: ['getUserPositionsComplete', address],
+    queryFn: async () => await getUserPositionsComplete(address),
+    enabled: !!address,
+  });
+
+  const filterActiveBets = useMemo(
+    () =>
+      userPositions?.filter(
+        userPosition => !userPosition.position.conditions[0].resolved
+      ) ?? [],
+    [userPositions]
+  );
+
+  const filterCompleteBets = useMemo(
+    () =>
+      userPositions?.filter(
+        userPosition => userPosition.position.conditions[0].resolved
+      ) ?? [],
+    [userPositions]
+  );
+
+  const { data: aiAgentsList } = useQuery({
+    queryKey: ['getAIAgents'],
+    queryFn: getAIAgents,
+    staleTime: Infinity,
+  });
+
+  const isAIAgent =
+    aiAgentsList?.some(aiAgent => String(aiAgent.address).toLowerCase() === address) ??
+    false;
+
+  if (!address || !isAddress(address)) return null;
+
+  const isLoadingUserPositions = userPositions === undefined;
+
+  const lossesInUSD = userPositions?.reduce((acc, userPosition) => {
+    const market = new Market(userPosition.market);
+
+    if (market.answer === null) return acc;
+
+    const amountSpentInUSD = tradesCollateralAmountUSDSpent({
+      fpmmTrades: userPosition.fpmmTrades,
+    });
+
+    return amountSpentInUSD + acc;
+  }, 0);
+
+  const winsInUSD = userPositions?.reduce((acc, userPosition) => {
```

Comment:
is this a total?

```suggestion
  const winsTotalInUSD = userPositions?.reduce((acc, userPosition) => {
```

something more descriptive would be better